tests/DontTest/SerializeTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DontTest;
+
+use Dont\Exception\NonSerializableObject;
+use Dont\Serialize;
+use DontTestAsset\NotSerializable;
+
+/**
+ * @covers \Dont\Serialize
+ */
+final class SerializeTest extends \PHPUnit_Framework_TestCase
+{
+    public function testWillThrowOnSerializationAttempt()
+    {
+        $object = new NotSerializable();
```

Comment:
Do we need the assignment? just `serialize(new NotSerializable());` on L22 should suffice